packages/components/src/box-control/axial-input-controls.tsx

```diff
@@ -129,25 +145,45 @@ export default function AxialInputControls( {
 						? selectedUnits.top
 						: selectedUnits.left;
 				return (
-					<UnitControl
-						{ ...props }
-						isFirst={ first === side }
-						isLast={ last === side }
-						isOnly={ only === side }
-						value={ [
-							parsedQuantity,
-							selectedUnit ?? parsedUnit,
-						].join( '' ) }
-						onChange={ createHandleOnChange( side ) }
-						onUnitChange={ createHandleOnUnitChange( side ) }
-						onFocus={ createHandleOnFocus( side ) }
-						onHoverOn={ createHandleOnHoverOn( side ) }
-						onHoverOff={ createHandleOnHoverOff( side ) }
-						label={ LABELS[ side ] }
-						key={ side }
-					/>
+					<HStack key={ `box-control-${ side }` }>
+						<FlexedBoxControlIcon side={ side } sides={ sides } />
+						<UnitControl
+							{ ...props }
+							isFirst={ first === side }
+							isLast={ last === side }
+							isOnly={ only === side }
+							value={ [
+								parsedQuantity,
+								selectedUnit ?? parsedUnit,
+							].join( '' ) }
+							onChange={ createHandleOnChange( side ) }
+							onUnitChange={ createHandleOnUnitChange( side ) }
+							onFocus={ createHandleOnFocus( side ) }
+							onHoverOn={ createHandleOnHoverOn( side ) }
+							onHoverOff={ createHandleOnHoverOff( side ) }
+							label={ LABELS[ side ] }
+							key={ side }
+						/>
+						<FlexedRangeControl
+							__nextHasNoMarginBottom
+							hideLabelFromVision
+							initialPosition={ 0 }
+							label={ LABELS[ side ] }
```

Comment:
I'd be curious to know what the best way is to label these and ensure they are associated together. 

I was looking at FontSizePicker since it is similar and saw the two labels there: 

https://github.com/WordPress/gutenberg/blob/83434bf46dc0529d9a97e8c4a5155d2a57c407f2/packages/components/src/font-size-picker/index.tsx#L219-L253

Initially, I didn't have the label on `RangeControl` but then I saw a violation in SB:

>Ensures every form element has a label
[More info...](https://dequeuniversity.com/rules/axe/4.7/label?application=axeAPI)
#inspector-range-control-19

So if there is a preferred alternative, we might want to note a follow-up for `FontSizePicker` as well.

Reply:
I followed up on this in another comment: https://github.com/WordPress/gutenberg/pull/56665/files#r1424628822